env/docker-compose.r2g.yaml

```diff
@@ -0,0 +1,64 @@
+version: "3.7"
+services:
+
+  micoo-nginx:
+    image: mattastach/micoo-nginx:latest
```

Comment:
I've been hosting this on my own Docker hub account, but can change this before merge if approved